tools/autograd/derivatives.yaml

```diff
@@ -514,6 +514,14 @@
 - name: log2(Tensor self) -> Tensor
   self: grad / (self * 0.6931471805599453)
 
+- name: logaddexp(Tensor self, Tensor other) -> Tensor
+  self: grad / (1 + exp(other - self))
+  other: grad / (1 + exp(self - other))
+
+- name: logaddexp2(Tensor self, Tensor other) -> Tensor
+  self: grad / (1 + pow(2, other - self))
```

Comment:
Isn't this one missing a `* 1/ln(2)` for the derivative of `log_2()`?

Reply:
not exactly, log2(x) = ln(x)/ln(2), so ∂/∂x log2(x) = 1/(x ln(2)), but for  ∂/∂x 2^x, your got another ln(2) in the numerator, so they cancel out. When in doubt, consult the [Wolfram Alpha](https://www.wolframalpha.com/input/?i=log2%282%5Ex+%2B+2%5Ey%29) ;)

Reply:
Ho right!
I was expecting something like that because the gradcheck is passing :D